src/mats-method-code/mats-method-code.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Get } from '@nestjs/common';
+import { ApiBadRequestResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+
+import { MatsMethodCodeDTO } from '../dto/mats-method-code.dto';
+import { MatsMethodCodeService } from './mats-method-code.service';
+
+@ApiTags('MATS Method Codes')
+@Controller('mats-method-code')
```

Comment:
should just be @Controller() with no prams. this will cause the route path to be /api/master-data-mgmt/mats-method-code\mats-method-codes becasue the routes.ts file specifies the path for each module.